src/plugins/news/screens/article/delegate.js

```diff
@@ -9,6 +13,31 @@ class Delegate extends AbstractDelegate {
       formName: formNames.ARTICLE,
     }, dependencies);
   }
+
+  handleSubmit(data, formDispatch, formProps) { // eslint-disable-line consistent-return
+    const { blocksmithState, dispatch } = this.component.props;
+    const { isValid, validEditorState } = validateBlocks(blocksmithState.editorState);
+
+    if (isValid) {
+      return AbstractDelegate.prototype.handleSubmit.call(this, data, formDispatch, formProps);
+    }
+
+    swal.fire({
+      title: 'Error saving article - one or more blocks is invalid.',
+      text: 'The invalid blocks cannot be saved. Would you like to save only the valid blocks?',
+      type: 'warning',
+      showCancelButton: true,
+      confirmButtonText: 'Yes',
+      confirmButtonClass: 'btn btn-danger',
+      cancelButtonClass: 'btn btn-secondary',
+    }).then((result) => {
+      if (!result.value) {
```

Comment:
@efjacobson - do we even need this check here? I mean, we can handle the `if (result.value)` case.

```
if (result.value) {
  dispatch(storeEditor(formProps.form, validEditorState));
  dispatch(submit(formProps.form));
}
```

Reply:
@arunkarnati works for me, updated